taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -66,6 +66,7 @@ def submit(
                 finished in asynchronous mode.
              timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished
                 before returning.
```

Comment:
```suggestion
                before returning.<br/>
```
Make sure the line break is also added in the other locations where the next line appears.